src/Compilers/Test/Core/Assert/ConditionalFactAttribute.cs

```diff
@@ -158,6 +158,7 @@ public static class ExecutionConditionUtil
 #endif
 
         public static bool IsWindows => Path.DirectorySeparatorChar == '\\';
+        public static bool IsWindows11OrGreater => IsWindows && Environment.OSVersion.Version.Build >= 22_000;
```

Comment:
Win10 and Win11 are still being updated SxS right? How does the build number work there?

Reply:
I'm unsure how the Win10 build numbers are being updated. I'm more than open to diff ways to check for Windows 11. 

One suggestion I got was to check for the path issue directly. Essentially don't check for windows 11, instead check for `aux.txt` path being escaped in `GetFullPath`. That is a more direct test but I worried about about other potential consequences there. 

I'm thinking of flipping to that for now and changing if we find such consequnecs